shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4

```diff
@@ -37,7 +37,7 @@ alterIndex
     ;
 
 dropTable
-    : DROP TABLE tableName (CASCADE CONSTRAINTS)? (PURGE)?
+    : DROP TABLE tableName
```

Comment:
I remember we add these key words in the previous PR, right?

Reply:
I've fixed this.